specs/auditbeat.spec.yml

```diff
@@ -1,43 +1,45 @@
-version: 2
-inputs:
-  - name: audit/auditd
-    description: "Auditd"
-    platforms: &platforms
-      - linux/amd64
-      - linux/arm64
-      - darwin/amd64
-      - darwin/arm64
-      - windows/amd64
-      - container/amd64
-      - container/arm64
-    outputs: &outputs
-      - elasticsearch
-      - kafka
-      - logstash
-      - redis
-    command:
-      args: &args
-        - "-E"
-        - "setup.ilm.enabled=false"
-        - "-E"
-        - "setup.template.enabled=false"
-        - "-E"
-        - "management.enabled=true"
-        - "-E"
-        - "logging.level=debug"
-        - "-E"
-        - "gc_percent=${AUDITBEAT_GOGC:100}"
-        - "-E"
-        - "auditbeat.config.modules.enabled=false"
-  - name: audit/file_integrity
-    description: "Audit File Integrity"
-    platforms: *platforms
-    outputs: *outputs
-    command:
-      args: *args
-  - name: audit/system
-    description: "Audit System"
-    platforms: *platforms
-    outputs: *outputs
-    command:
-      args: *args
+version: 2
+inputs:
+  - name: audit/auditd
+    description: "Auditd"
+    platforms: &platforms
+      - linux/amd64
+      - linux/arm64
+      - darwin/amd64
+      - darwin/arm64
+      - windows/amd64
+      - container/amd64
+      - container/arm64
+    outputs: &outputs
+      - elasticsearch
+      - kafka
+      - logstash
+      - redis
+    command:
+      args: &args
+        - "-E"
+        - "setup.ilm.enabled=false"
+        - "-E"
+        - "setup.template.enabled=false"
+        - "-E"
+        - "management.enabled=true"
+        - "-E"
+        - "logging.level=info"
```

Comment:
Is the switch from `debug` to `info` here intentional? Looks like it was set to debug on 8.5 as well https://github.com/elastic/elastic-agent/blob/6f08495275dba06a20aae3c8f62de69864893b90/internal/spec/auditbeat.yml#L8

Reply:
I see the same for Filebeat and Metricbeat, were we just default to debug logs being enabled all the time?

Reply:
Agreed, this is a good thing. I am more surprised we were defaulting logs to debug, I made this comment before I mentally worked through the consequences of this. There is likely a performance impact to defaulting to debug because of this.